CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing to @prezly/theme-kit-nextjs
+
+First off, thanks for taking the time to contribute! 🎉  
+All work on @prezly/theme-kit-nextjs happens directly on GitHub. Both core team members from Prezly and external contributors send pull requests which go through the same review process.
+
+---
+
+## Development
+
+Assuming you have cloned the repo and installed dependencies on your machine, the very next step is to develop the package.
+
+Since theme-kit provides a collection of base functions, hooks, and utilities that work with [Prezly themes](https://github.com/prezly?q=theme-nextjs&type=all&language=&sort=) under the hood, in order to make changes and test things out on your side, you're gonna need to keep a theme running in parallel with the theme-kit.
+
+So if you haven't already, go ahead and follow the [instructions on Bea theme](https://github.com/prezly/theme-nextjs-bea#quick-start) to get it up and running.
+
+As you can already see in the theme's `package.json` file, we're using `@prezly/theme-kit-nextjs` package. So we need to make sure that our theme is using our local version of theme-kit and not the one's in the theme's `node_modules` folder. How do we achieve that?
+
+### Option 1 - Using `npm link`
+
+In the **theme-kit's project root** folder run the following command:
+
+```
+npm link
+```
+
+Then go back to **theme's project root** and run:
+
+```
+npm link @prezly/theme-kit-nextjs
+```
+
+This will create a symbolic link between local version of the package and theme.
```

Comment:
Does `npm link` option even work? Because every time I've tried using it for testing a theme app with an updated content-renderer it never worked -- `npm` installs to versions of React breaking the build.

With `yalc` I had to struggle too, but eventually it worked.

Reply:
@e1himself For me it used to work at first, then for some unknown reason, it stopped working :) So I started using `yalc`. 

It's always nice to have a secondary option and it has less steps to set up I would say let's have it in place, however, I guess we can move it to 2nd option.

I have mentioned two different versions of react fix in the Common Errors section, I'll add a link to that.